pyke/targetpixelfile.py

```diff
@@ -95,25 +94,11 @@ def column(self):
     def row(self):
         return self.hdu['TARGETTABLES'].header['2CRV5P']
 
-    def plot(self, nframe=100, scale='linear', **kwargs):
+    def plot(self, nframe=100, **kwargs):
```

Comment:
I love this function!  Three comments:
* Let's make sure `plot()` has a short docstring.
* Can we rename `nframe` to `frame`?
* Can we support `cadence=XXX` as an alternative way to specify the desired frame?  (If set, it would use `frame = np.argwhere(cadence == data['CADENCENO'])`)